src/retriers.js

```diff
@@ -0,0 +1,14 @@
+export let retriers = {
+  is5xx: (response) => {
+    if(response && response.status &&
+      response.status >= 500 &&
+      response.status < 600) {
+        return true;
+    }
+    return false;
+  },
+
+  isNetworkError: (response) => {
+    return response instanceof Error;
```

Comment:
The way `json-fetch` is currently implemented, this would retry for all non-200 responses. Is there a specific attribute we could look for on the error to be more specific?